internal/uvm/create_lcow.go

```diff
@@ -199,6 +199,7 @@ func CreateLCOW(opts *OptionsLCOW) (_ *UtilityVM, err error) {
 						DefaultBindSecurityDescriptor: "D:P(A;;FA;;;SY)(A;;FA;;;BA)",
 					},
 				},
+				Plan9: &hcsschema.Plan9{},
```

Comment:
How did this work before when we did a hot add? I don't see that we ever added this device.

Reply:
Hyper-V was adding the vSMB device automatically, which includes the Plan9 device, when we were using initrd.

Reply:
Does a host side change remove this? I'm fine with it just trying to understand whats changing here

Reply:
I think this became necessary when we switched to vpmem root.